### ISSUE TITLE
"It's" instead of "its" in Clockwerk's Aghanim description

### ISSUE THREAD

**Issue by xvorkop** [ValveSoftware/Dota2-Gameplay#7816]
### Hero and spell

Clockwerk - Aghanim's Scepter

### Description

The description mentions "all enemies in it's range", while it should be "all enemies in its range". 

EDIT: Also, it should read "Supercharges Clockwerk's abilities" instead of "Supercharges Clockwerks abilities".

![photo_2023-04-01_17-08-21](https://user-images.githubusercontent.com/33763371/229305521-a17ef258-23b5-499d-8f0a-f434019896ee.jpg)

### Screenshots

![image](https://user-images.githubusercontent.com/33763371/229305449-18da6d74-4288-4ca2-8587-9c301d90382a.png)

**Comment by Aluerie** [comment]
#3477

**Comment by xvorkop** [comment]
> #3477

Yup, found it, too, although it features only one typo out of two.

Besides, it hasn't been answered, and the issue is still there.
